[2023-02-12T21:12:04.105+0200] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_process_sales.ds manual__2023-02-12T19:11:59.224964+00:00 [queued]>
[2023-02-12T21:12:04.111+0200] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_process_sales.ds manual__2023-02-12T19:11:59.224964+00:00 [queued]>
[2023-02-12T21:12:04.111+0200] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-02-12T21:12:04.111+0200] {taskinstance.py:1369} INFO - Starting attempt 1 of 4
[2023-02-12T21:12:04.111+0200] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-02-12T21:12:04.117+0200] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): ds> on 2023-02-12 19:11:59.224964+00:00
[2023-02-12T21:12:04.121+0200] {standard_task_runner.py:52} INFO - Started process 80480 to run task
[2023-02-12T21:12:04.124+0200] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'test_process_sales', 'ds', 'manual__2023-02-12T19:11:59.224964+00:00', '--job-id', '3628', '--raw', '--subdir', 'DAGS_FOLDER/process_sales/main.py', '--cfg-path', '/var/folders/1g/4w5x68gx3d92wgygb97jqbd40000gp/T/tmph7m_qj6x', '--error-file', '/var/folders/1g/4w5x68gx3d92wgygb97jqbd40000gp/T/tmpypksb3zf']
[2023-02-12T21:12:04.126+0200] {standard_task_runner.py:80} INFO - Job 3628: Subtask ds
[2023-02-12T21:12:04.155+0200] {task_command.py:371} INFO - Running <TaskInstance: test_process_sales.ds manual__2023-02-12T19:11:59.224964+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2023-02-12T21:12:04.193+0200] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=admin@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test_process_sales
AIRFLOW_CTX_TASK_ID=ds
AIRFLOW_CTX_EXECUTION_DATE=2023-02-12T19:11:59.224964+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-12T19:11:59.224964+00:00
[2023-02-12T21:12:04.194+0200] {python.py:173} INFO - Done. Returned value was: None
[2023-02-12T21:12:04.198+0200] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=test_process_sales, task_id=ds, execution_date=20230212T191159, start_date=20230212T191204, end_date=20230212T191204
[2023-02-12T21:12:04.228+0200] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-12T21:12:04.238+0200] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
